docs/INSTALLATION.md

```diff
@@ -1,15 +1,52 @@
 # Installation
 
-<!-- TODO: write document
+Gdscript is a domain specific language and can thereby requires the godot game engine to be installed.
```

Comment:
Is the local installation really required for Exercism to work with GDScript? As a person who never created a runner, I have no idea how they work ;) But I did create a docker configuration for GDScript runner, shouldn't that be sufficient?

Also, we should consistently call it `GDScript` :)

Reply:
Well, that would sorta mean that each student has to set up a docker environment. Which would force students to use docker desktop unless they are running linux or are running a vm. 

In general, I think that aint the idea. I can talk to Jeremy but I think the idea is that you download the language so you can use the languages cli.